PyDSS/dssInstance.py

```diff
@@ -92,6 +98,8 @@ def __init__(self, params):
             os.chdir(orig_dir)
         self._Logger.info('OpenDSS:  ' + reply)
 
+        data2 = self._dssInstance.YMatrix.GetPCInjCurr()
```

Comment:
I don't see this used anywhere.

Reply:
forgot to remove it. was doing some tests for DERMS implementation. Removing now